ieeg_epilepsy/sub-01/ses-post/ieeg/sub-01_ses-post_task-seizure_run-01_channels.tsv

```diff
@@ -0,0 +1,129 @@
+x'2	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'3	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'6	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'7	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'8	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'9	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'12	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'13	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+x'14	SEEG	uV	n/a	n/a	512	intracranial	x'	good
+e'1	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'2	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'3	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'4	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'5	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'8	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'9	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'10	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+e'11	SEEG	uV	n/a	n/a	512	intracranial	e'	good
+l'1	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'2	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'3	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'4	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'5	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'6	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'7	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'8	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+l'9	SEEG	uV	n/a	n/a	512	intracranial	l'	good
+f'1	SEEG	uV	n/a	n/a	512	intracranial	f'	bad
+f'2	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'3	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'4	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'8	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'9	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'10	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'11	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+f'12	SEEG	uV	n/a	n/a	512	intracranial	f'	good
+g'1	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'2	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'3	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'4	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'5	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'6	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'7	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'8	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'9	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'10	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'11	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+g'12	SEEG	uV	n/a	n/a	512	intracranial	g'	good
+o'1	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'2	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'5	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'6	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'10	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'11	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+o'12	SEEG	uV	n/a	n/a	512	intracranial	o'	good
+s'1	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+s'2	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+s'3	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+s'10	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+s'11	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+s'12	SEEG	uV	n/a	n/a	512	intracranial	s'	good
+fz	EEG	uV	n/a	n/a	512	unknown		good
+cz	EEG	uV	n/a	n/a	512	unknown		good
+ecg1	ECG	uV	n/a	n/a	512	unknown		good
+ecg2	ECG	uV	n/a	n/a	512	unknown		good
```

Comment:
you are missing a column for the last 5 rows, if it's not known, fill it with `n/a`

Reply:
(see also below for similar)